alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/test/java/org/alfresco/transformer/LibreOfficeTransformationIT.java

```diff
@@ -163,6 +187,11 @@ public LibreOfficeTransformationIT(final Pair<TestFileInfo, TestFileInfo> entry)
                 allTargets("quick.ods", spreadsheetTargets),
                 allTargets("quick.xls", spreadsheetTargets),
                 allTargets("quick.xlsx", spreadsheetTargets),
+                allTargets("quick.xml", spreadsheetTargets),
+                allTargets("quick.xltm", spreadsheetTargets),
+                allTargets("quick.xlam", spreadsheetTargets),
```

Comment:
This combination xlam --> pdf seems to be broken. Tested both manually and in a unit test against L/O or AIO containers, it throws the same JodConverter error we've seen in all the broken L/O transforms we removed.
`org.artofsolving.jodconverter.office.OfficeException: could not load document: source_12806769099172751483_quick.xlam`
I will add remove it from L/O config/tests for now and add it to the Jira part 2 ticket.

Reply:
This is done now

Reply:
Was xlam -> pdf in the supported transform config (either as part of PR # 246 or before) ?

Reply:
xlam > pdf was added here: https://github.com/Alfresco/alfresco-transform-core/pull/246/files#diff-ea090e9f38a105fca06b78337718d996R230